crates/biome_js_analyze/src/lint/nursery/no_array_constructor.rs

```diff
@@ -0,0 +1,96 @@
+use biome_analyze::{
+    context::RuleContext, declare_rule, Ast, FixKind, Rule, RuleDiagnostic, RuleSource,
+};
+use biome_console::markup;
+use biome_js_syntax::{AnyJsCallArgument, AnyJsExpression, JsCallArguments, JsExpressionStatement};
+use biome_rowan::AstNode;
+
+declare_rule! {
+    /// Disallow Array constructors.
+    ///
+    /// The corresponding ESLint rule:
+    /// https://eslint.org/docs/latest/rules/no-array-constructor#rule-details
```

Comment:
You can omit this, you already provided this information in the metadata. The website will take care of it for you.

Reply:
Thank you, omitted.